chemprop/cli/common.py

```diff
@@ -19,13 +21,9 @@ def add_common_args(parser: ArgumentParser) -> ArgumentParser:
     data_args.add_argument(
         "-r",
         "--reaction-columns",
-        nargs="+",
+    parser.add_argument("-g", "--gpu-spec", nargs="+", type=int, default=-1, help="indexes of GPUs to use (-1 for all (default), -2 for CPU)")
         help="The columns in the input CSV containing reactions.",
     )
```

Comment:
I think some artifacts happen here that insert the changes into the `add_argument` block

Reply:
Bleh okay I will do this the right way and just change my formatter haha